Tests/OctoKitTests/ReleasesTests.swift

```diff
@@ -11,12 +11,53 @@ import OctoKit
 
 final class ReleasesTests: XCTestCase {
     static var allTests = [
+        ("testListReleases", testListReleases),
         ("testPostRelease", testPostRelease),
         ("testLinuxTestSuiteIncludesAllTests", testLinuxTestSuiteIncludesAllTests)
     ]
 
     // MARK: Actual Request tests
 
+    func testListReleases() {
+        let session = OctoKitURLTestSession(expectedURL: "https://api.github.com/repos/octocat/Hello-World/releases", expectedHTTPMethod: "GET", jsonFile: "Fixtures/releases", statusCode: 200)
+        let task = Octokit().listReleases(session, owner: "octocat", repository: "Hello-World") {
+            switch $0 {
+            case let .success(releases):
+                XCTAssertEqual(releases.count, 2)
+                if let release = releases.first {
+                    XCTAssertEqual(release.tagName, "v1.0.0")
+                    XCTAssertEqual(release.commitish, "master")
+                    XCTAssertEqual(release.name, "v1.0.0 Release")
+                    XCTAssertEqual(release.body, "The changelog of this release")
+                    XCTAssertFalse(release.prerelease)
+                    XCTAssertTrue(release.draft)
+                    XCTAssertNil(release.tarballURL)
+                    XCTAssertNil(release.zipballURL)
+                    XCTAssertNil(release.publishedAt)
+                } else {
+                    XCTFail("Failed to unwrap `releases.first`")
+                }
+                if let release = releases.last {
+                    XCTAssertEqual(release.tagName, "v1.0.0")
+                    XCTAssertEqual(release.commitish, "master")
+                    XCTAssertEqual(release.name, "v1.0.0 Release")
+                    XCTAssertEqual(release.body, "The changelog of this release")
+                    XCTAssertFalse(release.prerelease)
+                    XCTAssertFalse(release.draft)
+                    XCTAssertNotNil(release.tarballURL)
```

Comment:
I just noticed, could we assert the correct URL here as you did with the Strings? Else this looks fine to me and we can merge it.

Reply:
Yes we could, and I fixed with using `XCTAssertEqual`.